packages/cli/lib/resources/template.html

```diff
@@ -10,8 +10,9 @@
 		<% if (htmlWebpackPlugin.options.manifest.theme_color) { %>
 			<meta name="theme-color" content="<%= htmlWebpackPlugin.options.manifest.theme_color %>">
 		<% } %>
+		<% const filesRegexp = htmlWebpackPlugin.options.inlineCss ? /\.(chunk\.\w{5}\.css|js)$/ : /\.(css|js)$/;%>
```

Comment:
`template.html` is already complicated and messed up. @developit any plans on making it similar to what CRA does?

Reply:
Well I can take a look at this in next iteration, but not for this PR